### PR TITLE
K8s: add real words to example values

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-2139
Preview:
- https://docs.redis.com/staging/DOC-2139/kubernetes/active-active/create-aa-crdb-cli/
- https://docs.redis.com/staging/DOC-2139/kubernetes/active-active/prepare-clusters/
- https://docs.redis.com/staging/DOC-2139/kubernetes/active-active/edit-clusters/
- https://docs.redis.com/staging/DOC-2139/kubernetes/active-active/global-config/
- https://docs.redis.com/staging/DOC-2139/kubernetes/active-active/edit-rerc/

### CODE REVIEW

File: content/kubernetes/active-active/create-reaadb.md [hunk 1]
```diff
@@ -22,37 +22,38 @@ Before creating an Active-Active database on Redis Enterprise for Kubernetes, yo
...
-      secretName: redis-enterprise-rerc1
+      recName: rec-chicago
+      recNamespace: ns-illinois
+      apiFqdnUrl: test-example-api-rec-chicago-ns-illinois.redis.com
+      dbFqdnSuffix: -example-cluster-rec-chicago-ns-illinois.redis.com
+      secretName: redis-enterprise-rerc-midway
     ```
 
-    Example RERC for a REC named `rec2` in the namespace `ns2`:
+    Example RERC (`rerc2raegan`) for the REC named `rec-arlington` in the namespace `ns-virginia`:
```
Comment (kaitlynmichael) [inline content/kubernetes/active-active/create-reaadb.md:44]:
```suggestion
    Example RERC (`rerc-raegan`) for the REC named `rec-arlington` in the namespace `ns-virginia`:
```

**Comment by kurtfm** [comment]
I think it it all generally looks good to me, but the only thing that caught my eye was that we went from using naming conventions of places and airports to `rerc3` at one point... and then in the edit phase we go back to using the flight motif.  The rerc3 is not wrong... but I guess I would stick with the flight motif.

**Comment by kaitlynmichael** [comment]
> I think it it all generally looks good to me, but the only thing that caught my eye was that we went from using naming conventions of places and airports to `rerc3` at one point... and then in the edit phase we go back to using the flight motif. The rerc3 is not wrong... but I guess I would stick with the flight motif.

Thanks for catching that @kurtfm ! I repalced the rerc1 and rerc2 but forgot that the edit clusters also uses a third cluster. It's fixed now.

File: content/kubernetes/active-active/edit-clusters.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ To communicate with other clusters, all participating clusters need access to th
     kubectl get secret -o yaml <rec-name>
     ```
 
-    This example shoes an admin credentials secret for an REC named `rec3`:
+    This example shoes an admin credentials secret for an REC named `rec-boston`:
```
Comment (rrelledge) [inline content/kubernetes/active-active/edit-clusters.md:37]:
```suggestion
    This example shows an admin credentials secret for an REC named `rec-boston`:
```

File: content/kubernetes/active-active/edit-clusters.md [hunk 3]
```diff
@@ -75,19 +75,19 @@ To communicate with other clusters, all participating clusters need access to th
 
 1. From one of the existing participating clusters, create a `RedisEnterpriseRemoteCluster` (RERC) custom resource for the new participating cluster.
 
-  This example shows a RERC custom resource for an REC named `rec3` in the namespace `ns3`. 
+  This example shows a RERC custom resource for an REC named `rec-boston` in the namespace `ns-massachusetts`.
```
Comment (rrelledge) [inline content/kubernetes/active-active/edit-clusters.md:78]:
```suggestion
  This example shows an RERC custom resource for an REC named `rec-boston` in the namespace `ns-massachusetts`. 
```

File: content/kubernetes/active-active/edit-clusters.md [hunk 4]
```diff
@@ -75,19 +75,19 @@ To communicate with other clusters, all participating clusters need access to th
...
-    dbFqdnSuffix: -example-cluster-rec3-ns3.redis.com
-    secretName: redis-enterprise-rerc3
+    recName: rec-boston
+    recNamespace: ns-massachusetts
+    apiFqdnUrl: test-example-api-rec-boston-ns-massachusetts.redis.com
+    dbFqdnSuffix: -example-cluster-rec-boston-ns-massachusetts.redis.com
+    secretName: redis-enterprise-rerc-logan
   ```
 
 1. Create the RERC custom resource.
```
Comment (rrelledge) [inline content/kubernetes/active-active/edit-clusters.md:93]: The numbered steps aren't rendering correctly in the "Create RERC" section. Adding whitespace before any code samples or extra sentences that belong to a step should fix this.

File: content/kubernetes/active-active/edit-clusters.md [hunk 5]
```diff
@@ -106,7 +106,7 @@ To communicate with other clusters, all participating clusters need access to th
 
   ```sh
   NAME        STATUS   SPEC STATUS   LOCAL
-  rerc3   Active   Valid         true
+  rerc-logan   Active   Valid         true
   ```
 
 ### Edit REAADB spec
```
Comment (rrelledge) [inline content/kubernetes/active-active/edit-clusters.md:112]: There's a similar issue with the numbered steps in "Edit REAADB spec".

File: content/kubernetes/active-active/global-config.md [hunk 6]
```diff
@@ -21,22 +21,22 @@ The [REAADB API reference](https://github.com/RedisLabs/redis-enterprise-operato
 
 1. Edit or patch the REAADB custom resource with your global configuration changes.
```
Comment (rrelledge) [inline content/kubernetes/active-active/global-config.md:22]: There are some formatting issues on this page. Most of the numbered steps aren't rendering correctly, and there are some formatting issues around the code block starting after "On each other participating cluster, check the secret status."

File: content/kubernetes/active-active/prepare-clusters.md [hunk 7]
```diff
@@ -25,7 +25,7 @@ An Active-Active database can span multiple clusters. Make sure you have enough
 
 The combination of the REC name and namespace (`<rec-name>.<namespace-name>`) must be unique for each participating cluster the Active-Active database.
```
Comment (rrelledge) [inline content/kubernetes/active-active/prepare-clusters.md:26]: Missing word?
```suggestion
The combination of the REC name and namespace (`<rec-name>.<namespace-name>`) must be unique for each participating cluster in the Active-Active database.
```

File: content/kubernetes/active-active/prepare-clusters.md [hunk 8]
```diff
@@ -25,7 +25,7 @@ An Active-Active database can span multiple clusters. Make sure you have enough
...
 
-For example, if you have two K8s clusters, each with their own REC named `rec1` in a namespace named `ns1`. The value of `<rec-name>.<namespace-name>` for both RECs would be `rec1.ns1`. These can't be used for the same Active-Active database.
+For example, if you have two K8s clusters, each with their own REC named `rec-chicago` in a namespace named `ns-illinois`. The value of `<rec-name>.<namespace-name>` for both RECs would be `rec-chicago.ns-illinois`. These can't be used for the same Active-Active database.
```
Comment (rrelledge) [inline content/kubernetes/active-active/prepare-clusters.md:28]: Sentence fragment fix
```suggestion
For example, if you have two K8s clusters, each with their own REC named `rec-chicago` in a namespace named `ns-illinois`, the value of `<rec-name>.<namespace-name>` for both RECs would be `rec-chicago.ns-illinois`. These can't be used for the same Active-Active database.
```

File: content/kubernetes/active-active/prepare-clusters.md [hunk 9]
```diff
@@ -36,7 +36,6 @@ Active-Active databases require external routing access to sync properly. To con
 
 The admission controller using a validating webhook to dynamically validate resources configured by the operator. The `ValidatingWebhookConfiguration` is required for Active-Active databases. Learn how to enable and configure admission controller in the [Enable admission controller]({{<relref "/kubernetes/deployment/quick-start.md#enable-the-admission-controller/">}}) section of the [Deploy Redis Enterprise Software for Kubernetes]({{<relref "/kubernetes/deployment/quick-start.md">}}) instructions.
```
Comment (rrelledge) [inline content/kubernetes/active-active/prepare-clusters.md:37]:
```suggestion
The admission controller uses a webhook to dynamically validate resources configured by the operator. The `ValidatingWebhookConfiguration` is required for Active-Active databases. Learn how to enable and configure the admission controller in the [Enable admission controller]({{<relref "/kubernetes/deployment/quick-start.md#enable-the-admission-controller/">}}) section of the [Deploy Redis Enterprise Software for Kubernetes]({{<relref "/kubernetes/deployment/quick-start.md">}}) instructions.
```

**Review by rrelledge — CHANGES_REQUESTED** [review]
Looks good to me aside from some typos and formatting issues.

**Review by cmilesb — APPROVED** [review]

**Pull request merged**